frame/instrumental-strategy/src/mock/runtime.rs

```diff
@@ -1,16 +1,16 @@
+use crate as pallet_instrumental_strategy;
 use frame_support::{parameter_types, traits::Everything, PalletId};
 use frame_system::{EnsureRoot, EnsureSigned};
 use orml_traits::parameter_type_with_key;
-use primitives::currency::{CurrencyId, ValidateCurrencyId};
+use pallet_collective::EnsureProportionAtLeast;
+use primitives::currency::CurrencyId;
```

Comment:
You accidentally deleted `ValidateCurrencyId` import. Could you return it, please?
> error[E0412]: cannot find type `ValidateCurrencyId` in this scope